README.md

```diff
@@ -1,20 +1,16 @@
+[![Kotlin](https://img.shields.io/badge/Kotlin-1.5.31-blue.svg?style=flat&logo=kotlin)](http://kotlinlang.org)
+[![Maven Central](https://img.shields.io/maven-central/v/fr.acinq.lightning/lightning-kmp)](https://search.maven.org/search?q=g:fr.acinq.lightning%20a:lightning-kmp*)
+![Github Actions](https://github.com/ACINQ/lightning-kmp/actions/workflows/test.yml/badge.svg)
 [![License](https://img.shields.io/badge/license-Apache%202.0-blue.svg)](LICENSE)
 
-**lightning-kmp** is a [Kotlin Multiplatform](https://kotlinlang.org/docs/multiplatform.html) implementation of the Lightning Network optimized for mobile wallets.
-It can run on many different platforms, including mobile devices (iOS and Android).
-
-This software implements the [Lightning Network Specifications (BOLTs)](https://github.com/lightningnetwork/lightning-rfc).
+**lightning-kmp** is a [Kotlin Multiplatform](https://kotlinlang.org/docs/multiplatform.html) implementation of the Lightning Network (see the [Lightning Network Specfications (BOLTs)](https://github.com/lightning/bolts)) optimized for mobile wallets. It can run on many different platforms, including mobile devices (iOS and Android).
```

Comment:
```suggestion
**lightning-kmp** is a [Kotlin Multiplatform](https://kotlinlang.org/docs/multiplatform.html) implementation of the Lightning Network (see the [Lightning Network Specifications (BOLTs)](https://github.com/lightning/bolts)) optimized for mobile wallets. It can run on many different platforms, including mobile devices (iOS and Android).
```